README.md

```diff
@@ -1,19 +1,98 @@
+5. Log into the guest VM with either ssh or the GUI. The default username/password is hydro:hydro
+6. From the root directory `/home/hydro`, clone this repository into the hydroshare folder
+7. `cd` into the hydroshare folder and run `./hsctl rebuild --db` to build the application and run it
+8. If all goes well, your local version of Hydroshare should be running at http://192.168.56.101:8000
+
+For more detailed installation, please see this document: [Getting Started with HydroShare](https://github.com/hydroshare/hydroshare/wiki/getting_started)
+
+## Usage
+
+For all intents and purposes, Hydroshare is a large Python/Django application with some extra features and technologies added on:
+- SOLR for searching
+- Redis for caching
+- RabbitMQ for concurrency and serialization
+- iRODS for a federated file system
+- PostgreSQL for the database backend
+
+#### The `hsctl` Script
+
+The `hsctl` script is your primary tool in interacting with and running tasks against your Hydroshare install. It has the syntax `./hsccl [command]` where `[command]` is one of:
+
+- `loaddb`: Deletes existing database and reloads the database specified in the `hydroshare-config.yaml` file.
+- `managepy [args]`: Executes a `python manage.py [args]` call on the running hydroshare container.
+- `maint_off`: Removes the maintenance page from view (only if NGINX is being used).
+- `maint_on`: Displays the maintenance page in the browser (only if NGINX is being used).
+- `rebuild`: Stops, removes and deletes only the hydroshare docker containers and images while retaining the database contents on the subsequent build as defined in the `hydroshare-config.yaml` file
+- `rebuild --db`: Fully stops, removes and deletes any prior hydroshare docker containers, images and database contents prior to installing a clean copy of the hydroshare codebase as defined in the `hydroshare-config.yaml` file.
+- `rebuild_index`: Rebuilds the solr/haystack index in a non-interactive way.
+- `restart`: Restarts the django server only (and nginx if applicable).
+- `start`: Starts all containers as defined in the `docker-compose.yml` file (and nginx if applicable).
+- `stop`: Stops all containers as defined in the `docker-compose.yml` file.
+- `update_index`: Updates the solr/haystack index in a non-interactive way.
+
+#### Testing and Debugging
+
+Tests are run via normal Django tools and conventions. However, you should use the `hsctl` script mentioned abouve with the `managepy` command. For example: `./hsctl managepy test hs_core.tests.api.rest.test_resmap --keepdb`.
+
+There are currently over 600 tests in the system, so it is highly recommended that you run the test suites separately from one another.
+ 
+#### Local iRODS
+
+Local iRODS is _not_ required for development unless you are specifically working on the iRODS integration. However, if you want to work with iRODS or you simply want to learn about it, you can enable it locally by following these steps:
+- Coming soon
```

Comment:
To the maximal extent possible avoid "coming soon" as too often soon never comes.  I suggest for now not promising steps until there are some:

However,if you want to work with iRODS or you simply want to learn about it, you can enable it locally.